src/GridFS/Bucket.php

```diff
@@ -587,6 +589,24 @@ public function openUploadStream(string $filename, array $options = [])
         return fopen($path, 'w', false, $context);
     }
 
+    /**
+     * For legacy applications that needs to open stream from a string, the bucket can be registered in the stream
+     * wrapper to be used by default when no GridFS context is provided.
+     * Supported file name must follow this pattern:
+     *     gridfs://<bucket-alias>/<filename>
```

Comment:
`unlink` is not supported ATM.

Reply:
> `unlink` is not supported ATM.

Oh, you're correct. We don't define an `unlink()` method on the stream wrapper, so that wouldn't be handled at all.

Reply:
I created PHPLIB-1323